UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/IFulfilmentFileShareService.cs

```diff
@@ -9,5 +9,6 @@ public interface IFulfilmentFileShareService
     {
         Task<List<FulfillmentDataResponse>> QueryFileShareServiceData(List<Products> products);
         Task<string> UploadFileShareServiceData(string uploadFileName, List<FulfillmentDataResponse> fulfillmentDataResponse, string storageAccountConnectionString, string containerName);
+        Task DownloadFileShareServiceFiles(SalesCatalogueServiceResponseQueueMessage message, List<FulfillmentDataResponse> fulfillmentDataResponses, string exchangeSetRootPath);
```

Comment:
Although I understand that Fulfilment (English) can also be spelt Fulfillment (American) it would be better if we could be consistent and adopt the English spelling.

Reply:
did changes and one of the files name is changes